src/saving/SaveManagerGUI.cs

```diff
@@ -203,23 +203,23 @@ private void DeleteSelectedButtonPressed()
     {
         GUICommon.Instance.PlayButtonPressSound();
 
-        deleteSelectedConfirmDialog.DialogText =
+        deleteSelectedConfirmDialog.GetNode<Label>("Label2").Text =
             string.Format(CultureInfo.CurrentCulture,
                 TranslationServer.Translate("DELETE_SELECTED_SAVE_WARNING"),
                 Selected.Count);
-        deleteSelectedConfirmDialog.PopupCenteredMinsize();
+        deleteSelectedConfirmDialog.PopupCentered(deleteSelectedConfirmDialog.GetMinimumSize());
```

Comment:
I have one idea regarding this. Couldn't we make an extension method for this? Perhaps like PopupCenteredProperly? That could then just do this GetMinimumSize call automatically making all of the places using it look neater and more future proof against needing more changes.

Reply:
Yeah, that's what I was originally going for (with messy implementation) but finally found that this also work with lesser code to write. But I agree that an extension should just be used, I think `PopupCenteredShrink` name is more fitting.